python/sglang/srt/managers/router/scheduler.py

```diff
@@ -27,44 +27,35 @@ def get_priority_queue(self, forward_queue):
             return forward_queue
         elif self.schedule_heuristic == "fcfs":
             return forward_queue
-        elif self.schedule_heuristic == "weight":
+        elif self.schedule_heuristic == "dfs-weight":
             last_node_to_reqs = defaultdict(list)
             for req in forward_queue:
                 last_node_to_reqs[req.last_node].append(req)
             for node in last_node_to_reqs:
                 last_node_to_reqs[node].sort(key=lambda x: -len(x.prefix_indices))
```

Comment:
Is it possible for reqs with the same last node to have different len(prefix_indices)? Does the order matter?